google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -161,8 +161,7 @@ appengine.deployment.flexible.nogaesupport=No modules with Google App Engine fle
 appengine.deployment.error.cancelled=Deployment process was cancelled.
 appengine.deployment.staging.artifact.as=Staging artifact as: {0}
 appengine.deployment.error.sdk.still.installing=Google Cloud SDK with App Engine Java needs to be completely installed to perform this action. Please wait for installation to complete.
-appengine.deployment.error.sdk.not.available=Google Cloud SDK is not available.
-appengine.deployment.error.sdk.invalid=Google Cloud SDK is invalid.
+appengine.deployment.error.sdk.not.available=Deployment failed: Google Cloud SDK is not ready. Please try again and allow Google Cloud SDK installation to complete.
```

Comment:
Wondering if this is clearer:

> Deployment failed: Google Cloud SDK is not ready. Please try again and allow Google Cloud SDK installation to complete

to

`Deployment failed: Google Cloud SDK is not ready. Please wait for the Google Cloud SDK installation to complete and try again`

Reply:
Most probable scenario here is that user cancelled the SDK installation and that led to deployment / local run cancel - in this case there's basically nothing to wait for *until* one runs deploy / run again and install restarts / completes. This could be a bit misleading then?

Reply:
oh I see. I misinterpreted the scenario then. sounds good